go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/cosmos/cosmos-sdk
 
 require (
+	github.com/Masterminds/glide v0.13.2
```

Comment:
Why were all these deps added?

Reply:
I was sure I had removed this, my fault. I will fix this in an ensuing `go mod tidy` PR ASAP.

Reply:
thank you @alessio